setup.py

```diff
@@ -119,9 +116,13 @@
 # For Python3.4, this is OSX 10.6, but we need Thread Local Support (__thread)
 if 'darwin' in sys.platform and PY3:
   mac_target = sysconfig.get_config_var('MACOSX_DEPLOYMENT_TARGET')
+  # The working computer's mac os x version defines what frameworks are
+  # available to build against.
+  mac_version = platform.mac_ver()[0]
```

Comment:
Arguably, it *should* be able to (heck, it builds against 10.6 sans `__thread`), but for whatever reason I kept getting tripped up by missing headers.